dev-overrides.yaml

```diff
@@ -0,0 +1,11 @@
+modules:
+      repositories:
+          - name: cct_module
+            git:
+                  url: https://github.com/luck3y/cct_module.git
```

Comment:
luck3y?

Reply:
Yeah, I went back and forth on these, on the one hand, it requires --overrides=dev-overrides.yaml when building, OTOH, not defaulting them to the working branch makes it irrelevant if it points to master on both.  I'll update to take those out.